content/applications/websites/website/web_design/building_blocks.rst

```diff
@@ -128,20 +152,18 @@ Depending on the visibility settings, some elements can become hidden from your
 make a building block visible again, go to the :guilabel:`Invisible Elements` section at the bottom
 of the configurator and select a building block.
 
-Saving a custom building block
-==============================
-
-You can save a customized building block and reuse it elsewhere. To do so, select it, navigate to
-the :guilabel:`Customize` tab, and click the floppy disk icon to save it.
-
-.. image:: building_blocks/saving-custom-block.png
-   :alt: Saving a building block
+Mobile view customization
```

Comment:
I'd like to avoid repeating the same info in the next paragraph, like "open the website editor, click the mobile icon," etc. I suggest replacing the paragraph with something along the following lines:

You can customize building block elements for the mobile view without impacting the desktop view. To do so, open the website editor, click the :icon:`fa-mobile` (:guilabel:`mobile`) icon at the top, and select the building block element. Then, you can:

- reorder the elements by clicking the :icon:`fa-angle-left` :icon:`fa-angle-right` (:guilabel:`left/right arrow`) icons;
- edit the the :ref:`Cols <building_blocks/cols>` and :ref:`Visibility <building_blocks/visibility>` features in the :guilabel:`Customize` tab of the website editor.

[end of suggestion] 

- I feel like "tailor" is a little bit outdated in this context
- With this paragraph structure, I suggest having the part where we explain how to reorder elements before the one about Cols and Visibility since we're already in the left part of the screen after saying "and select the building block element" and it flows better that way IMO. I feel like if we explain the Cols and Visibility features first, people might look for the arrows to reorder the elements in the website editor as well. I don't know if this makes sense, let me know if it doesn't!

Reply:
sounds good to me, thank you